### ISSUE TITLE
Scan/Rescan UI-triggered job node selector and tolerations

### ISSUE THREAD

**Issue by ltartarini** [robusta-dev/robusta#851]
Hi *, I installed Robusta on my K8s (EKS-based) cluster, from the UI when I try to scan or rescan the cluster, it triggers a Kubernetes Job which creates a Pod that won't be scheduled in any cluster Node due to lack of `nodeSelector` and `tolerations`.

```
kubectl get job                                         
NAME                         COMPLETIONS   DURATION   AGE
krr-job-1683283651-2795317   0/1           5s         5s

kubectl get pod                                                                                                      
NAME                                 READY   STATUS    RESTARTS      AGE
krr-job-1683283651-2795317-7qglt     0/1     Pending   0             23s
```

I would like to set such configs into the `values.yaml` file similar to what happen for bot `kubewatch` (https://github.com/robusta-dev/robusta/blob/master/helm/robusta/values.yaml#L406) and `runner` (https://github.com/robusta-dev/robusta/blob/master/helm/robusta/values.yaml#L438).

**Comment by RoiGlinik** [comment]
@ltartarini We are working on that, will be added to next release.

**Comment by RoiGlinik** [comment]
https://github.com/robusta-dev/robusta/pull/906

**Comment by ltartarini** [comment]
Awesome, thanks!

**Comment by ltartarini** [comment]
When the next release will be distributed? 😄

**Comment by arikalon1** [comment]
within a week or so :)

**Comment by arikalon1** [comment]
Released as part of 0.10.17
Closing it
